src/lib/expansion/expansion.spec.ts

```diff
@@ -129,13 +129,30 @@ describe('MatExpansionPanel', () => {
 
       const arrow = fixture.debugElement.query(By.css('.mat-expansion-indicator')).nativeElement;
 
-      expect(arrow.style.transform).toBe('rotate(0deg)', 'Expected no rotation.');
+      expect(arrow.style.transform).toBe('rotate(45deg)', 'Expected 45 degree rotation.');
 
       fixture.componentInstance.expanded = true;
       fixture.detectChanges();
       tick(250);
 
-      expect(arrow.style.transform).toBe('rotate(180deg)', 'Expected 180 degree rotation.');
+      expect(arrow.style.transform).toBe('rotate(225deg)', 'Expected 225 degree rotation.');
+    }));
+
+  it('should update the indicator position when the position is set programmatically',
```

Comment:
Also add a test that verifies that the position still works when it's set on the accordion?

Reply:
Also added a test for displayMode since it looks like that was missing in the tests as well.